integr8ly/integreatly-operator
# INTLY-6303-fix-amq-memory-usage-alert

# Description
When memory usage in `api-server` container in AMQ namespace is increased above 90% of capacity, `AMQOnlinePodHighMemory` alert is not being triggered. The expression defining the trigger for the alert needed to be modified as it referred to a label that did not exist.

Also, an unnecessary alert RHSSOPodHighMemory has been removed

JIRA: https://issues.redhat.com/browse/INTLY-6303
          https://issues.redhat.com/browse/INTLY-5491
 
# Verification
- Open terminal on `api-server` container and use a bash script to get memory utilisation to 90%+
- Wait 15 minutes 
- Verify Alert fires
- Ensure RHSSOHighPodMemory Alert is no longer showing
## Type of change


## Checklist
